src/bindings/html/dom.js

```diff
@@ -77,19 +76,31 @@ export function translateMutations(view, langs, mutations) {
 
   const elements = [];
 
-  targets.forEach(target => target.childElementCount ?
-      elements.push(...getTranslatables(target)) : elements.push(target));
+  targets.forEach(target => {
+    if (target.childElementCount) {
+      elements.push(...getTranslatables(target));
+    } else {
+      if (target.hasAttribute('data-l10n-id')) {
```

Comment:
the ternary style would require me to have an if in the else structure. I feel like we're paying with code readability for dense coding at some point. if/else block may be longer but I feel like it's more readable. Would you agree?